plugins/PagePerformance/Columns/TimeNetwork.php

```diff
@@ -41,6 +41,10 @@ public function onNewAction(Request $request, Visitor $visitor, Action $action)
             return false;
         }
 
+        if ($networkTime < 0) {
+            throw new \Exception(sprintf('Value for %1$s can\'t be negative.', $this->getRequestParam()));
```

Comment:
Might need to through an `InvalidRequestParameterException` exception so it appears in the tracking failure log plus I think it might have some other advantages like a more correct response code etc

Reply:
could maybe move that method in a `base` class that all the performance dimensions share?

Reply:
done